shop/src/app/components/presentational/products-details-view/products-details-view.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+import { ProductDetails } from 'src/app/modules/shared/types/product-details.types';
+
+@Component({
+  selector: 'app-products-details-view',
+  templateUrl: './products-details-view.component.html',
+  styleUrls: ['./products-details-view.component.scss']
+})
+export class ProductsDetailsViewComponent {
+  @Input() product: ProductDetails | undefined;
```

Comment:
Can you maybe add the optional `?` marker for this Input?

Reply:
I have since then made it an Observable of the type, so am I right to assume it is no longer needed?

Reply:
I would assume it's ok like that as well